packages/programs/src/programs/ChestProgram.sol

```diff
@@ -13,6 +13,11 @@ contract ChestProgram is Hooks.ITransfer, DefaultProgram {
   constructor(IBaseWorld _world) DefaultProgram(_world) { }
 
   function onTransfer(Hooks.TransferContext calldata ctx) external view onlyWorld {
-    require(_isAllowed(ctx.target, ctx.caller), "Only approved callers can transfer to/from the chest");
+    if (_isProtected(ctx.target)) {
```

Comment:
wasn't sure if this should be just a force field check or a force field + energy check

I was thinking this same logic could apply to spawn tiles and maybe beds